src/component/MovieCard.js

```diff
@@ -38,21 +68,16 @@ class MovieCard extends React.Component {
                 <br/>
               </Typography>
               <Typography style={classes.views} variant="body2" color="textSecondary" component="p">
-                  {'views: ' + this.props.movie.num_of_visits}
-
+                  {'Views: ' + this.props.movie.num_of_visits} 
+                  <br></br>     
+                  {this.markWatched()}
```

Comment:
izmjeni funkciju da ti govori da li je watched i u odnosu na to rendaj tekst ili ne